.env.example

```diff
@@ -15,4 +15,4 @@ REACT_APP_PORTIS_KEY=b2b7586f-2b1e-4c30-a7fb-c2d1533b153b
 
 REACT_APP_ZAPPER_API_KEY=96e0cc51-a62e-42ca-acee-910ea7d2a241
 REACT_APP_GRAPH_API_URL=https://api.thegraph.com/subgraphs/name/kibagateaux/dd-test
-
+REACT_APP_GRAPH_CHAINLINK_FEED_REGISTRY_API_URL=https://api.thegraph.com/subgraphs/name/hepnerthomas/chainlink-feed-registry-test
```

Comment:
@kibagateaux We need to add this second subgraph URL into the `.env` file on Vercel and on the team's local machines.

Reply:
@kibagateaux In addition, I am planning to move the [Chainlink Feed Registry subgraph deployed on mainnet](https://thegraph.com/hosted-service/subgraph/hepnerthomas/chainlink-feed-registry-test) into the debtdao/subgraph repo given that we'll likely be using more than one subgraph going forward. Any issue with that?